hydrolib/core/dflowfm/tim/serializer.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import Iterable, List, Optional
+
+from hydrolib.core.basemodel import SerializerConfig
+from hydrolib.core.dflowfm.ini.io_models import Datablock, DatablockRow
+from hydrolib.core.dflowfm.ini.serializer import MaxLengths, SectionSerializer
+from hydrolib.core.dflowfm.tim.parser import TimTimeSerie
+
+
+class TimSerializerConfig(SerializerConfig):
+    datablock_indent: int = 4
+    datablock_spacing: int = 2
+
+
+class TimTimeSeriesSerializer(SectionSerializer):
```

Comment:
I am not sure if inheriting from the `SectionSerializer` would be a good idea since that is a INI specialized class.
In my opinion, it is fine if you use the stuff that you need from the `SectionSerializer` and put it here, because INI and TIM are just not the same file formats and thus the same serializer should not be used. 

Maybe we can look it this together.